src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -121,6 +121,32 @@ The features below are only available at the page level. (You can't use them ins
 This is because these features modify the [Response headers](https://developer.mozilla.org/en-US/docs/Glossary/Response_header), which can't be modified after they're sent to the browser. In SSR mode, Astro uses HTML streaming to send each component to the browser as it renders them. This makes sure the user sees your HTML as fast as possible, but it also means that by the time Astro runs your component code, it has already sent the Response headers.
 :::
 
+### `Astro.cookies`
+
+On the `Astro` global, this interface allows to manipulate cookies. It allows to directly get them from requests and set them in responses. Whereas `Astro.request.headers.get('cookie')` gets all cookies and still need to be parsed, `Astro.cookies.get` gets a single cookie in an object.
+
+More details about the API in the reference [Astro.cookies](/en/reference/api-reference/#astrocookies)
+
+The example below shows how to update the value of a counter cookie.
+
+```astro title="src/pages/index.astro" {4,10}
+---
+let counter = 0
+
+const cookie = Astro.cookies.get("counter")
+
+if(cookie.value != undefined){
```

Comment:
Just checking here, because admittedly I'm not familiar with using this, but would this be an opportunity to write something using `Astro.cookies.has()` so we can showcase more of the API?

Reply:
yes, sure, I even hesitated about that and mentioned it. Now I added it and pushed, also updated highlights.